roles/make-applier-projects-unique/tasks/update-galaxy-requirements.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: "Set regexp_replace_pairs for galaxy requirements"
+  replace:
+    path: "{{ item.galaxy_requirements }}"
+    regexp: "{{ pair.regexp }}"
+    replace: "{{ pair.replace }}"
+  with_items:
+  - "{{ galaxy_regexp_replace_pairs }} "
```

Comment:
let's add ` | default([])` in this variable to avoid it bumming out if this variable isn't defined